src/modules/epsrManager/process.cpp

```diff
@@ -25,33 +25,47 @@ bool EPSRManagerModule::setUp(ModuleContext &moduleContext, Flags<KeywordBase::K
 // Run main processing
 Module::ExecutionResult EPSRManagerModule::process(ModuleContext &moduleContext)
 {
-    auto &moduleData = moduleContext.dissolve().processingModuleData();
-
-    std::map<std::string, EPData> potentials;
+    if (averagingLength_)
+        Messenger::print("Potentials will be averaged over {} sets (scheme = {}).\n", averagingLength_.value(),
+                         Averaging::averagingSchemes().keyword(averagingScheme_));
+    else
+        Messenger::print("Potentials: No averaging of potentials will be performed.\n");
 
+    auto &moduleData = moduleContext.dissolve().processingModuleData();
+    // Potential Set
+    PotentialSet potentials;
     // Loop over target data
```

Comment:
Just for neatness I'd shuffle these lines around
```suggestion

    // Loop over target data and form summed / averaged potentials
    PotentialSet potentials;
```